projects/pumpbtc/index.js

```diff
@@ -7,11 +10,33 @@ const config = {
   ethereum: { owners: ['0x1fCca65fb6Ae3b2758b9b2B394CB227eAE404e1E', '0x3d9bCcA8Bc7D438a4c5171435f41a0AF5d5E6083'], tokens: ['0xC96dE26018A54D51c097160568752c4E3BD6C364', ADDRESSES.ethereum.WBTC], },
   bsc: { owners: ['0x2Ee808F769AB697C477E0aF8357315069b66bCBb'], tokens: [ADDRESSES.bsc.BTCB], },
   mantle: { owners: ['0xd6Ab15B2458B6EC3E94cE210174d860FdBdd6b96'], tokens: ['0xC96dE26018A54D51c097160568752c4E3BD6C364'], },
+  bitcoin: {},
 }
 
-Object.keys(config).forEach(chain => {
+Object.keys(config).forEach(async chain => {
   const { owners, tokens, } = config[chain]
-  module.exports[chain] = {
-    tvl: async (api) => api.sumTokens({ owners, tokens })
+
+  if (chain === 'bitcoin') {
+    module.exports[chain] = {
+      tvl:
+        async (api) => {
+          const addresses = await getConfig('pumpbtc', undefined, { fetcher: getStakingAddresses })
+          return sumTokens({ api, owners: addresses })
+        }
+    }
+  } else {
+    module.exports[chain] = {
+      tvl: async (api) => api.sumTokens({ owners, tokens })
+    }
   }
 })
+
+async function getStakingAddresses() {
+  let res = await utils.fetchURL('https://dashboard.pumpbtc.xyz/api/dashboard/btc/realtime')
+
+  const btcAddresses = res.data.data.details.btc.details.filter(i => +i.amount > 0).map(d => d.address)
+  const babylonAddresses = res.data.data.details.babylon.details.filter(i => +i.amount > 0).map(d => d.address)
+  return [... new Set(btcAddresses.concat(babylonAddresses))]
```

Comment:
hi @xianshui I noticed that addresses are duplicated between btc & babylon, so refactored to remove duplicates.

Checked a few addresses randomly, the value match the one returned by our api?

Reply:
hi @g1nt0ki,  thanks for the modification. I checked some addresses both in btc address list and babylon address list, the values for the btc address list match. But the values for the babylon address list does not match, for instance the actual balance of an address is 0.05, but the platform api sumTokens returns 0, that made us can not count the balance staked in babylon chain. This is the point we need to handle, could we  return the total amount of the babylon part  through our API? Or is there any other better suggested method to do this?

Reply:
@xianshui sorry, missed this comment, sorry to keep bothering you but can you update the api to return the babylon addresses as well? 
We are caching api response, so, would be fine even if api goes down. but if rely on number from api, we are lost if api becomes inaccessible

Reply:
hi @g1nt0ki , we update the api to return the babylon addresses, please have a check and merge the PR, thanks.